### PR TITLE
Merge `develop` into `master`

### PR DESCRIPTION
**Pull request by passalis**
This PR aims to synchronize recent `develop` changes into `master`. 

This PR also bumps `av` version from `8.0.1` to `8.0.3` to resolve installation failure, as well as a minor fix to HR pose estimation which didn't download all necessary files.

### CODE REVIEW

**Comment by passalis** [comment]
I think I have resolved the issues. If this passes, then I think we can merge it, in order to start preparing the new release.

**Comment by passalis** [comment]
This is ready on my side.

File: Dockerfile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 FROM ubuntu:20.04
 
-ARG branch=master
+ARG branch=develop
```
Comment (tsampazk) [inline Dockerfile:3]: Is this intented?

File: Dockerfile-cuda [hunk 2]
```diff
@@ -1,6 +1,6 @@
 FROM nvidia/cuda:11.2.2-cudnn8-devel-ubuntu20.04
 
-ARG branch=master
+ARG branch=develop
```
Comment (tsampazk) [inline Dockerfile-cuda:3]: .

File: Dockerfile-embedded [hunk 3]
```diff
@@ -186,7 +186,7 @@ RUN echo 'source /opt/ros/${ROS_DISTRO}/setup.bash' >> /root/.bashrc
 CMD ["bash"]
 WORKDIR /
 
-RUN git clone --depth 1 --recurse-submodules -j8 --branch master https://github.com/opendr-eu/opendr
+RUN git clone --depth 1 --recurse-submodules -j8 --branch develop https://github.com/opendr-eu/opendr
```
Comment (tsampazk) [inline Dockerfile-embedded:189]: .

**Review by tsampazk — COMMENTED** [review]
Thank you @passalis! I just have some doubts about some references to develop.

File: Dockerfile-embedded (hunk 3 above)
Comment (passalis) [reply Dockerfile-embedded]: Good catch, i fixed them.

**Review by tsampazk — APPROVED** [review]
Looks good to me, thank you!

**Review by Pavlos-Tosidis — APPROVED** [review]
Looks good

**Pull request merged**